spec/jobs/google_scraping_job_spec.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe GoogleScrapingJob, type: :job do
+  include ActiveJob::TestHelper
+
+  describe '#perform' do
+    it 'enqueues google scraping job' do
+      user = Fabricate(:user)
+      keyword = Fabricate(:keyword, user_id: user[:id], keyword: 'AWS')
+
+      expect do
+        GoogleScrapingJob.perform_later(keyword.id, keyword.keyword)
+      end.to have_enqueued_job(GoogleScrapingJob)
+
+      assert_enqueued_with(job: GoogleScrapingJob, args: [keyword.id, 'AWS'])
+    end
+
+    it 'creates GoogleScrapingService instance' do
```

Comment:
seems the test doesn't have anything relatives to `creates GoogleScrapingService instance` it seems returns a `Keyword instance`

Reply:
I think I use the incorrect test case name, Maybe should be `performs GoogleScrapingService`
Fixed in 219bcdce3e0bfaeb1520cf80234cc1b3238bdabb